service.subtitles.rvm.addic7ed/addon.xml

```diff
@@ -4,7 +4,7 @@
   version="3.1.8"
   provider-name="Roman V.M.">
 <requires>
-  <import addon="xbmc.python" version="2.25.0"/>
+  <import addon="xbmc.python" version="3.0.0"/>
```

Comment:
No need because matrix submission is handled automatically by addon-submitter script.